### PR TITLE
Fixes #7210: Remove dependency on hub in rudder-dev

### PR DESCRIPTION
**Pull request by peckpeck**
https://www.rudder-project.org/redmine/issues/7210

### CODE REVIEW

**Comment by peckpeck** [comment]
PR updated

**Comment by peckpeck** [comment]
PR updated

File: scripts/rudder-dev/rudder-dev [hunk 1]
```diff
@@ -577,6 +580,8 @@ def get_config(item, error):
 
 # Get github user as used by the hub command
 def get_hub_user():
```
Comment (VinceMacBuche) [inline scripts/rudder-dev/rudder-dev]: maybe the function should be renamed to

get_github_user

File: scripts/rudder-dev/rudder-dev [hunk 2]
```diff
@@ -589,6 +594,8 @@ def get_hub_user():
 
 # Get github token as used by the hub command
 def get_hub_token():
```
Comment (VinceMacBuche) [inline scripts/rudder-dev/rudder-dev]: same here!

**Comment by peckpeck** [comment]
PR updated

**Pull request merged**